ddl/backfilling.go

```diff
@@ -550,24 +538,12 @@ func getBatchTasks(t table.Table, reorgInfo *reorgInfo, kvRanges []kv.KeyRange,
 	} else {
 		prefix = t.RecordPrefix()
 	}
-	// Build reorg tasks.
-	job := reorgInfo.Job
 	//nolint:forcetypeassert
 	phyTbl := t.(table.PhysicalTable)
-	jobCtx := reorgInfo.d.jobContext(job.ID)
-	for i, keyRange := range kvRanges {
+	for _, keyRange := range kvRanges {
 		taskID := taskIDAlloc.alloc()
 		startKey := keyRange.StartKey
 		endKey := keyRange.EndKey
-		endK, err := getRangeEndKey(jobCtx, reorgInfo.d.store, job.Priority, prefix, keyRange.StartKey, endKey)
```

Comment:
This seems to lead to performance regression mentioned in https://github.com/pingcap/tidb/issues/26683. @zimulala do we have relevant integration test about this issue?

Reply:
Let me take a look at this problem.